src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/queue/InboundQueueServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound.queue;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.jms.core.MessageCreator;
+import uk.nhs.digital.nhsconnect.lab.results.mesh.message.InboundMeshMessage;
+import uk.nhs.digital.nhsconnect.lab.results.mesh.message.WorkflowId;
+import uk.nhs.digital.nhsconnect.lab.results.utils.ConversationIdService;
+import uk.nhs.digital.nhsconnect.lab.results.utils.JmsHeaders;
+import uk.nhs.digital.nhsconnect.lab.results.utils.TimestampService;
+
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import java.time.Instant;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class InboundQueueServiceTest {
```

Comment:
JUnit 5-ify - no need for `public`
```suggestion
class InboundQueueServiceTest {
```

Reply:
Thanks!